styles/style-blog-post.css

```diff
@@ -0,0 +1,310 @@
+@import url("https://fonts.googleapis.com/css2?family=Overpass+Mono&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Source+Code+Pro:wght@300&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Francois+One&display=swap");
+/* body styles */
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  margin: 0em;
+  padding: 0em;
+  background-color: #f2f2f2;
+}
+/* header title */
+#header {
+  padding-top: 2vh;
+  width: 100%;
+  display: flex;
+  flex-flow: column;
+  justify-content: space-evenly;
+  align-items: center;
+}
+#header > * {
+  margin: 0em;
+}
+#header hr {
+  color: transparent;
+  line-height: 1px;
+  padding: 1px;
+  margin-top: 5vh;
+  width: 100%;
+  border-bottom: 1px solid gray;
+  border-top: 1px solid gray;
+  opacity: 50%;
+}
+#title {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 50%;
+  color: #5a6c73;
+}
+#title h1 {
+  margin-bottom: 0.4em;
+  font-family: "Overpass Mono", monospace;
+  font-size: 1.5em;
+  color: #f98770;
+  letter-spacing: 0.4em;
+}
+#title h5 {
+  margin: 0em;
+  font-family: "Open Sans", sans-serif;
+}
+#header h4 {
+  color: #5a6c73;
+  margin-top: 4vh;
+  font-family: "Open Sans", sans-serif;
+}
+
+button {
+  background-color: #f98770;
+  color: white;
+  font-family: "Francois One", sans-serif;
+  box-shadow: none;
+  border: none;
+}
+#head-button {
+  margin: 5vh;
+}
+
+/* top button */
+#top-button {
+  position: sticky;
+  top: 35%; 
+  border-radius: 50%;
+  background-color: #f98770;
+  height: 2.5em;
+  width: 2.5em;
+  font-size: 1em;
+  margin: 0em;
+  margin-left: -75%;
+align-self: flex-start;
+}
```

Comment:
I like the addition of the consistent "back to top" button across the website pages.